content/php/concepts/string-functions/terms/strlen/strlen.md

```diff
@@ -13,19 +13,35 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-The **`strlen()`** method returns the number of bytes in a given string.
+The **`strlen()`** method returns the length of a `string`.
```

Comment:
I think it's worth having a note about bytes like the one in the [PHP documentation](https://www.php.net/manual/en/function.strlen.php#:~:text=//%207%0A%3F%3E-,Notes,-%C2%B6).
```suggestion
The **`strlen()`** method returns the length of a `string`.

> **Note:** This method returns the number of bytes rather than the number of characters.

```